### ISSUE TITLE
[Enhancement] Windows: add menu bar

### ISSUE THREAD

**Issue by frehner** [kitematic/kitematic#860]
Forgive me if I missed this, but I searched this repo and didn't find anything.

On OSX, the menu bar provides access to things such as "About Kitematic" (which looks great with that new PR, btw), "Preferences", etc. However, the Windows version doesn't appear to have any equivalent... Unless I'm missing something?

**Comment by FrenchBen** [comment]
Unfortunately I don't use the Windows GUI, but the menu Preferences in OSX, can also be found within the Kitematic GUI - If you look at the bottom left of the Kitematic window, you can see a small 'gear' icon, which will show you the preferences window.

**Comment by frehner** [comment]
Good to know, thanks.  So that's a workaround for preferences... everything else in the OSX menu bar is still inaccessible though (or very well hidden).

**Comment by FrenchBen** [comment]
Probably won't work - but did you try to just press 'Alt' while Kitematic is open? In windows that usually forces any/all menu to appear

**Comment by frehner** [comment]
Yeah, I think I tried that and nothing happened --  but I'm away from my work/windows environment for the weekend to be 100% sure.

**Comment by frehner** [comment]
Just to clarify, the ALT button doesn't do anything.  Also, another reason why this is a slight annoyance for Windows users: the only way to tell which version Kitematic is currently on is by going to the "Uninstall or Change a program" window and looking at the version there. 

This was tested on both 0.7.6 and 0.8.0-rc4.

**Comment by pmario** [comment]
IMO this is a bug and not an enhancement. There is code, that creates a menu. ... I just doesn't work on windows. see: https://github.com/docker/kitematic/blob/master/src/app.js#L54

**Comment by Bigguy34** [comment]
Ok I started to fix this, check out this [PR](https://github.com/docker/kitematic/pull/1559)
